pkg/scheduler/scheduler_test.go

```diff
@@ -894,10 +895,27 @@ func Test_UnionedGVKs(t *testing.T) {
 			},
 			enableInPlacePodVerticalScaling: true,
 		},
+		{
+			name:    "plugins with default profile (queueingHint: enabled)",
+			plugins: schedulerapi.PluginSet{Enabled: defaults.PluginsV1.MultiPoint.Enabled},
+			want: map[framework.GVK]framework.ActionType{
+				framework.Pod:                   framework.Add | framework.UpdatePodLabel | framework.UpdatePodScaleDown | framework.UpdatePodTolerations | framework.UpdatePodSchedulingGatesEliminated | framework.Delete,
```

Comment:
This new test shows that we eliminated all Pod/Update events from in-tree plugins.